blockly/src/utils/BlocklyCommands.java

```diff
@@ -477,4 +492,23 @@ public static void turnEntity(Entity entity, Direction direction) {
     // so the player can not glitch inside the next tile
     pc.position(oldP);
   }
+
+  /**
+   * Checks whether the boss's view direction equals the given direction.
+   *
+   * <p>This method is specifically used for the boss in Blockly Chapter 3, Level 4.
+   *
+   * @param direction the direction to check against
+   * @return {@code true} if the boss's view direction equals the given direction; {@code false} if
+   *     the direction does not match, or if the boss or its PositionComponent is missing
+   */
+  public static boolean checkBossViewDirection(Direction direction) {
+    return Game.allEntities()
+        .filter(e -> e.name().equals("Blockly Black Knight"))
```

Comment:
Ich stolper hier grade etwas.
Der String ist an dieser stelle ist MAGIC und den findet hier später niemand, aber du kommst hier auch nicht anders an den namen von dem vieh....

Kannst du in `BlocklyMonster` einen konstanten String "public static final BOSS_NAME="Blockly Black Knight" erstellen und dann im Enum eintragen und hier verwenden?